osquery/tables/system/darwin/firewall.cpp

```diff
@@ -94,12 +95,16 @@ QueryData parseALFExceptionsTree(const pt::ptree& tree) {
     if (it.second.get("alias", "").length() > 0) {
       std::string path;
       auto alias_data = it.second.get<std::string>("alias", "");
+      auto status = pathFromNestedPlistAliasData(alias_data, path);
 
-      if (pathFromPlistAliasData(alias_data, path).ok()) {
-        r["path"] = path;
-        r["state"] = INTEGER(it.second.get("state", -1));
-        results.push_back(r);
+      if (!status.ok()) {
+        TLOG << status.getMessage();
```

Comment:
I think we should add some context to the error message: `"Could not parse nested plist for applications: " << status.getMessage()`. Or something more accurate compared to my example. ;)